app/admin/featured_projects.rb

```diff
@@ -0,0 +1,12 @@
+ActiveAdmin.register FeaturedProject do
+  menu parent: 'Events'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.